pkgs/applications/networking/cluster/k3s/update.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl gnugrep gnused jq
+
+set -eou pipefail
```

Comment:
This works, but it feels like it shouldn't.
To just set pipefail, it's `set -o pipefail`, so I expect in the short form the `o` has to be last, right before pipefail.
I ran this locally, and the invocation here does work, but I think `set -euo pipefail` or `set -eu -o pipefail` is clearer.

A quick `rg 'set.*pipefail'` shows that both those forms are in use in nixpkgs too, among others.

Reply:
Done